api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -796,6 +796,130 @@
 			<column name="uuid" type="char(38)" />
 		</addColumn>
 	</changeSet>
+	<changeSet author="vshankar" id="201609171146-2">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cohort_member" columnName="cohort_member_id"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="cohort_member_id" type="int">
+				<constraints nullable="false"/>
+			</column>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-3">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cohort_member" columnName="start_date"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="start_date" type="datetime">
+				<constraints nullable="false"/>
+			</column>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-4">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cohort_member" columnName="end_date"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="end_date" type="datetime"/>
+		</addColumn>
+	</changeSet>
+	<changeSet author="vshankar" id="201609171146-5">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="cohort_member" columnName="creator"/>
+			</not>
+		</preConditions>
+		<addColumn tableName="cohort_member">
+			<column name="creator" type="int">
+				<constraints nullable="false"/>
+			</column>
+		</addColumn>
+		<addForeignKeyConstraint baseColumnNames="creator" baseTableName="cohort_member" constraintName="cohort_member_creator" deferrable="false" initiallyDeferred="false" referencedColumnNames="user_id" referencedTableName="users"/>
```

Comment:
Sam here, you want the add constraint in separate changeset otherwise this will fail for installations where they have existing rows in this table. You will also have to first populate the column for existing rows